packages/ckeditor5-show-blocks/tests/showblockscommand.js

```diff
@@ -40,7 +42,94 @@ describe( 'ShowBlocksCommand', () => {
 		it( 'should be a command instance', () => {
 			expect( command ).to.be.instanceOf( Command );
 		} );
+
+		it( 'should set "#affectsData" property to false', () => {
+			expect( command.affectsData ).to.be.false;
+		} );
+
+		it( 'should set "#value" property to false', () => {
+			expect( command.value ).to.be.false;
+		} );
 	} );
 
-	describe( '#execute()', () => {} );
+	describe( '#execute()', () => {
+		it( 'should set "ck-show-blocks" class on the root when executed for the first time', () => {
+			editor.execute( 'showBlocks' );
+
+			expect( editor.editing.view.document.roots.get( 'main' )._classes.has( 'ck-show-blocks' ) ).to.be.true;
```

Comment:
Please don't access `private` properties in tests unless absolutely necessary. If we had 500 tests like this, a single change in the internal implementation of a view `Element` would require a maaaaaaassive test rewrite. There's a https://ckeditor.com/docs/ckeditor5/latest/api/module_engine_view_element-Element.html#function-hasClass for that.